### ISSUE TITLE
Use of Timepoint Metadata

### ISSUE THREAD

**Issue by jogiles** [broadinstitute/wot#69]
Hello!

First, thank you for developing a great pseudotime tool.

We have a time course: day 0, 8, 15, 30. From prior knowledge, we know there are terminal cells at day 8, 15, and 30. There should be branding points that that end at day 8 and others that go on to the later timepoints. When we calculate wot trajectory, all cells at day 8 have an intermediate trajectory value.

Is it possible for this method to detect terminal cells at intermediate time points? Is there any way to indicate this in function arguments?

Thank you!!!!
Jo

**Comment by geoffschieb** [comment]
Dear Jo,

Could you please elaborate on what you mean by "terminal cells at
intermediate time-points"? What's the mathematical definition of this? I
think you mean cells that are 100% fated and essentially identical to their
final fate (like at the bottom of a Waddington valley). Or do you mean
cells with no descendants (like "terminally ill")?

Best,
Geoff

**Comment by jogiles** [comment]
Hello Geoff,

Thank you for the speedy response!

Terminal cells -- meaning -- we know from experimental evidence that there are cells at intermediate time points (day 8) that have no progeny and they will eventually die. It is experimentally known -- not based on a mathematic definition. 

Also, they are the most different transcriptionally and epigenetically compared to the root/progenitor cell(s) at timepoint 0 -- this is known from sorting experiments to subset the cells then performing bulk RNA and ATAC-seq.

I hope this helps. 

Thank you!!!
Jo

**Comment by geoffschieb** [comment]
Yes, I would call "have no progeny and will eventually die" a precise
mathematical definition :-)

Our method is specifically designed to handle this through the "growth
rates". This parameter specifies the expected number of descendants each
cell should have at the next time point. So if a cell will have a 50%
chance of dying, then it would give rise to 0.5 descendants. If a cell does
not divide or die, it is its own "descendant".

We compute transport maps in Notebook 2 of the Waddington OT tutorial. The
growth rate parameter is defined for reprogramming in Step 1 and used in
Step 2.

To define the parameter, keep in mind that the program takes a rate, and
then computes a number of descendants depending on the gap between time
points. So for example if your time-points are in units of "days" then a
cell dividing every 1 day would give rise to 2 descendants after 1 day. If
your time points are 2 days apart then there would be 2 divisions so 4
descendants. The rate would be 2 per day. In the case of death, a rate of
0.5 per day is interpreted as a "half-life" like in radioactivity.

Best,
Geoff
